live/app/templates/password-reset.hbs

```diff
@@ -1,4 +1,10 @@
 <div class="password-reset-page">
+
+  <div class="password-reset-page__inscription">
+    <p class="password-reset-page__inscription-text" title="">Vous n'avez pas encore de compte Pix ?</p>
+    <a class="password-reset-page__inscription-button" href="/inscription">S'inscrire <span class="sr-only"> sur Pix</span> </a>
```

Comment:
- mettre un `{{link-to}}`
- ajouter un "petit" TA qui vérifie qu'on a bien le lien